.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test on ${{ matrix.os }} with Rust ${{ matrix.rust }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        rust:
+          - 1.21.0
```

Comment:
I'd personally prefer to not test MSRV unless necessary, I think it's fine to stick to stable/beta/nightly.

Can builders on windows and macos also be added?

Reply:
I added the MSRV because it was already tested on Travis. But sure, I removed it now (while rebasing).

#27 mentions problems with macOS and Windows:

- macOS fails the smoke test from time to time. I don't have a machine to reproduce and debug this locally, that's why I left it out of CI config
- Windows build for i686-pc-windows-gnu can't find a linker. Yet again, I don't have a machine and can't debug. But I kept the one job that you added to main.yml.